CHANGES.txt

```diff
@@ -18,6 +18,8 @@ Breaking Changes
 Changes
 =======
 
+ - Add index setting ``translog.durability``.
```

Comment:
Should be in Fixes not Changes, because this setting is needed in order for the ``translog.sync_interval` `to take effect. So In the Fixes I would write something like:

Added missing table setting ``translog.durability`` which should be set accordingly so that  ``translog.sync_interval`` can take effect.